src/main/java/com/github/alexcojocaru/mojo/elasticsearch/LoadElasticSearchUtility.java

```diff
@@ -0,0 +1,177 @@
+package com.github.alexcojocaru.mojo.elasticsearch;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.http.HttpEntityEnclosingRequest;
+import org.apache.http.client.ClientProtocolException;
+import org.apache.http.client.methods.*;
+import org.apache.http.entity.ContentType;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.logging.Log;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * @author gfernandes
+ */
+public class LoadElasticSearchUtility {
```

Comment:
Moved logic to load the ES with data from the file as a utility class instead of a mojo so I was able to use it from two places without repeating code.